Cinema Room Manager/task/src/cinema/Menu.java

```diff
@@ -26,7 +26,11 @@ public void displayMenu() {
                     System.out.println(roomView);
                     break;
                 case 2:
-                    room.buyTicket();
+                    System.out.println("Enter a row number:");
```

Comment:
refactoring should not change logic of the code and you changed it.
previously there was a loop with validation and some hints for user to understand why inputed data are incorrect and user was able to buy only valid ticket.

you should validate seat position inside buy ticket method and rise an exception - this is the most popular solution solution

good alternative is to use Result monad. Read more here:
https://medium.com/@afcastano/monads-for-java-developers-part-2-the-result-and-log-monads-a9ecc0f231bb

Reply:
Simple solution is to create to validation methods in Room class:
* isSeatValid(rowNum, seatNum)
* isSeatAvailable(rowNum, seatNum)

Reply:
```java
        bool seatEntered = false;
        while (!seatEntered) {
            System.out.println("Enter a row number:");
            selectedRowNumber = sc.nextInt();
            System.out.println("Enter a seat number in that row:");
            selectedSeatNumber = sc.nextInt();
            if (!room.isSeatValid(selectedRowNumber, selectedSeatNumber)) {
                System.out.println("Wrong input");
            } else if (!room.isSeatAvailable(selectedRowNumber, selectedSeatNumber)) {
                System.out.println("That ticket has already been purchased!");
            } else {
                seatEntered = true;
            }
        };
        room.buyTicket(selectedRowNumber, selectedSeatNumber);
```

Reply:
Seat validation is set in class Menu. Logic of the code is now restore.